Makefile

```diff
@@ -47,44 +43,45 @@ default: help
 .PHONY: docker/build
 ## Build the docker image
 docker/build:
-	@docker build -t ${DOCKER_HUB_REPO}:latest -t ${DOCKER_HUB_REPO}:${DOCKER_IMAGE_TAG} .
+	docker build -t build-tools:latest .
```

Comment:
why would you change this to build-tools instead of using the valid repository name straight away?

Reply:
because we build locally and tag with real repo in `docker/tag` target

Reply:
fixed